GUI/Properties/Resources.resx

```diff
@@ -7116,6 +7116,7 @@ Open settings now?</value></data>
   <data name="MainWaitPleaseWait" xml:space="preserve"><value>Please wait</value></data>
   <data name="MainWaitDone" xml:space="preserve"><value>All done!</value></data>
   <data name="ManageKspInstancesNotValid" xml:space="preserve"><value>Directory {0} is not valid KSP directory.</value></data>
```

Comment:
(Another complaint about old code) Can you please change to, "is not **a** valid KSP directory" ?